src/domain/event/occurrences/Occurrences.tsx

```diff
@@ -374,11 +374,20 @@ const OccurrenceInfo: React.FC<OccurrenceInfoProps> = ({
     </>
   );
 
+  const showAllLocationEvents = () => {
+    const url = `/?places=${placeId || eventLocationId}`;
+    window.open(url);
+  };
```

Comment:
Do we want to open this in a new window? If so, button might need text for screen reader that says link opens in a new window.

Reply:
I don't know, do we? I chose this approach so the user who is enrolling does not lost the enrolment page he was in. Should I change it to `location.href=url`? Would that be better?

Reply:
This will be caught in next accessibility audit if we open in a new tab without warning the user (screen reader). 

We should as Pirjo / Joakim about this :)

Reply:
Asked from Joakim. If there can be a button inside the Link-component without wrecking styles I can use it. This doc page says there can https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-function-component.

Reply:
Great, seems like HDS forwards the ref which is required with that:

https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/components/button/Button.tsx#L80

Reply:
yes, and the passHref was an important parameter.